src/IO/H5/File.hpp

```diff
@@ -102,6 +107,9 @@ class H5File {
     return h5::get_group_names(file_id_, "/");
   }
 
+  /// Get the InputSource.yaml string embedded in the file
+  const std::string input_source() const;
```

Comment:
`const std::string` -> `std::string` (only need the `const` if it were a reference)

Reply:
Addressed comments from @nilsdeppe and @wthrowe and squashed